src/main/resources/swaggerTemplateOverloads/generatedAnnotation.mustache

```diff
@@ -3,4 +3,6 @@
 {{^hideGenerationTimestamp}}
 @javax.annotation.Generated(value = "{{generatorClass}}", date = "{{generatedDate}}")
 {{/hideGenerationTimestamp}}
-// GS annotation goes here
+{{#apiInformation}}
```

Comment:
this is always true, right? I would remove the variable and reduce the complexity a tiny bit